packages/runner/lib/sdk/sdk.ts

```diff
@@ -263,6 +215,58 @@ export class NangoSyncRunner extends NangoSyncBase {
     sendLogToPersist = NangoActionRunner['prototype']['sendLogToPersist'];
     logAPICall = NangoActionRunner['prototype']['logAPICall'];
 
+    public async setMergingStrategy(merging: 'ignore_if_modified_after' | 'override', ...models: string[]): Promise<void> {
```

Comment:
sugg: I would use a json params, it's not clear to understand after the fact especially with cryptic record name
![Screenshot 2025-01-31 at 11 06 35](https://github.com/user-attachments/assets/a2871b10-2d2f-4901-bb7e-36e32fadf186)

Reply:
you mean 
```
public async setMergingStrategy({merging, models}: {merging: 'ignore_if_modified_after' | 'override', models: string[]})
```
I initially choose to stay close to the interface of `batchSave` with a first argument being the value to save and the second one the model, in the case of setMergingStrategy being able to pass more than one. Using a object as a param would mean models must be an array which is slight less clean since in most of cases it would be a single model. `await nango.setMergingStrategy({merging: `ignore_if_modified_after`, models: [IdTooLarge])

Reply:
Ah that's a good point, if we are forward-thinking then reversing the model name could help typing and autocompletion with the change I have done in the hackathon:
- `await nango.setMergingStrategy(modelName: TModel, strategy: union); ` 

 - We could allow the first param to be an array too. 
 `await nango.setMergingStrategy(modelName: TModel | TModel[], strategy: union); ` 

--- 

If we go JSON you could allow models to be a string or array. 

Side note: the code is not doing parallelism so it's not different than having N calls. We could not care about multi-model syncs (which is already an exception rather than the norm), and only allow a string. So if you have multiple models you "just" call this multiple times.

Reply:
So I'm not sure tbh, I don't have strong opinion, I just feels like it can be confusing to have free-form strings like this

Reply:
> We could allow the first param to be an array too.
> `await nango.setMergingStrategy(modelName: TModel | TModel[], strategy: union); `

This is where my brain immediately went as well.

Reply:
> We could allow the first param to be an array too.
> await nango.setMergingStrategy(modelName: TModel | TModel[], strategy: union);